src/autocomplete.js

```diff
@@ -340,7 +380,7 @@ var Autocomplete = function() {
         if (!selected || !this.editor || !this.popup.isOpen)
             return this.hideDocTooltip();
         this.editor.completers.some(function(completer) {
-            if (completer.getDocTooltip)
+            if (completer.getDocTooltip && selected.completerId === completer.id)
                 doc = completer.getDocTooltip(selected);
```

Comment:
usage of a result of `getDocTooltip()` that is defined to have a void return in àce.d.ts

Reply:
Changed definition of `getDocTooltip` to fit actual code